modules/dev2plus/hw_01/src/Tree/Tree.js

```diff
@@ -0,0 +1,16 @@
+const Node = require("../Node/Node");
+
+class Tree {
+  constructor(name = "") {
+    this._name = name;
```

Comment:
using underscore at the beginning of the property name is not a good idea, was your idea to differentiate "private" values? if so then it's better to actually make them private :)

Reply:
Yes, it was an older custom... :) I will fix it.